README.md

```diff
@@ -0,0 +1,147 @@
+
+# Proposal
+
+I propose a channel protocol which steals the main idea from Aumayr et al. while retaining the time-lock precedence of BOLT 3 transactions.
+That is, absolute time-locks are settled before relative time-locks to avoid having to account for relative time-locks when calculating absolute time-locks.
+
+The guiding principal is to keep commitment transactions symmetric but to use the asymmetric knowledge of the parties to simulate the way lightning works right now.
+The main benefits of this seem to be:
+
+- It is more elegant as there are half the number of possible transactions. I expect this will follow through to reduced implementation complexity and maybe make it easier to explain as well.
+- Every output can just be a 2-of-2 multi-sig (which could be a single key with MuSig or OP_CHECKMULTISIG with ECDSA etc). This makes the transactions a little smaller even when using 2-of-2 with ECDSA on balance (I think). The space saving comes from avoiding explicit revocation keys in the commitment transaction outputs.
+- The design works today and easily transitions into a post-taproot world although there is still a bit to explore there [5].
+
+## Commitment transaction outputs
+
+The way the fund transaction is constructed and negotiated remains unchanged.
+For each commitment transaction parties exchange two curve points each:
+
+- A _publication_ point (denoted P) whose secret is revealed if that party broadcasts the commitment transaction.
+- A _revocation_ point (denoted R) whose secret is explicitly revealed to the other party when revoking this commitment transaction.
+
+I'll use the notation (Pa, Ra) and (Pb, Rb) to refer to the publication and revocation points of Alice and Bob respectively and 2-of-2(X,Y) to refer to a multi-signature output that supports adaptor signatures and requires the owners of X and Y to authorise it (e.g. `OP_CHECKMULTISIG` with ECDSA [6], or MuSig with Schnorr etc [7]).
+
+The scriptPubkey of all outputs on a commitment transaction is 2-of-2(Pa + Ra, Pb + Rb). 
+For a slight increase in fungibility you can imagine that we pseudorandomly randomize each output so that they don't all look identical to blockchain observers.
+
+When signing a commitment transaction, the parties instead exchange adaptor signatures such that the party who publishes the commitment transaction must reveal their publication secret.
+This means that if a party broadcasts a commitment transaction for which they have already revealed their revocation secret the other party will know both secrets and therefore both secret keys in 2-of-2(Pa + Ra, Pb + Rb).
+They can use this to unilaterally spend from all the outputs on the commitment transaction to punish the malicious party.
+
+Note there is no problem with both parties broadcasting the same commitment transaction at the same time with their different signatures.
```

Comment:
> Isn't the problem that only one of the transactions can be included?

The point of this sentence is to say this isn't a problem.

>  This means that both parties need to monitor their node's mempool to learn about the other party's transaction and publication secret.

They don't need to but if they do they can settle the state _more_ efficiently than they otherwise would be able to .